cmd/origin-version-change/main.go

```diff
@@ -43,10 +43,18 @@ func isYAML(data []byte) bool {
 }
 
 func changeTemplateObjectsVersion(t *templateapi.Template) {
-	if errs := kruntime.DecodeList(t.Objects, api.Scheme); len(errs) > 0 {
+	changeObjects(t.Objects)
+}
+
+func changeListObjectsVersion(t *api.List) {
```

Comment:
i don't this there is a point to have this one line function... just call changeObjects()